package.json

```diff
@@ -68,34 +67,21 @@
     "lint:fix": "eslint $npm_package_config_jsfiles --fix"
   },
   "dependencies": {
-    "@wordpress/autop": "^1.0.6",
-    "@wordpress/compose": "^1.0.1",
-    "@wordpress/deprecated": "^1.0.0-alpha.2",
-    "@wordpress/hooks": "^1.2.1",
-    "@wordpress/i18n": "^1.1.0",
-    "@wordpress/is-shallow-equal": "^1.0.1",
+    "@wordpress/blocks": "^1.0.2",
```

Comment:
I wasn't aware of how they are consumed. I assumed that everything is pulled in from npm to avoid the dance with dependencies and keeping them in sync.